packages/store/package.json

```diff
@@ -13,7 +13,7 @@
     "prepare": "yarn build && chmod u+x git-install.sh",
     "git:install": "bash git-install.sh",
     "codegen": "ts-node ./scripts/codegen.ts",
-    "tablegen": "../cli/dist/mud.js tablegen",
+    "tablegen": "mud tablegen",
```

Comment:
i'd prefer keeping the local version for this script, so it works in the CI when we do changes to tablegen (without releasing a new version)

Reply:
what if, instead, we have CI use a locally linked mud?

Reply:
That would work too, we just have to set it up. If it "just works" I'm down to do that instead in this PR, but wouldn't lose too much time on it, we can always change it in a follow up

Reply:
will change it back for easy and add an issue to come back to this

Reply:
https://github.com/latticexyz/mud/issues/489